apps/verticalhome/js/pin_app_navigation.js

```diff
@@ -55,25 +78,16 @@
 
       this.selectedElemIndex = 0;
 
-      for (var elemIndex in this.elements) {
-        this.elemList.appendChild(this.elements[elemIndex]);
-      }
+      this.elements.forEach(function(elem, index, array){
```

Comment:
you can safely omit unused parameters and write something like this:
``this.elements.forEach( (elem) => { this.elemList.appendChild(elem); });``

But it's a bit unclear for me, why do we need a copy of elements array?